yarn.lock

```diff
@@ -10385,6 +10385,16 @@ react-custom-scrollbars@4.2.1:
     prop-types "^15.5.10"
     raf "^3.1.0"
 
+react-datetime@2.16.3:
+  version "2.16.3"
+  resolved "https://registry.yarnpkg.com/react-datetime/-/react-datetime-2.16.3.tgz#7f9ac7d4014a939c11c761d0c22d1fb506cb505e"
+  integrity sha512-amWfb5iGEiyqjLmqCLlPpu2oN415jK8wX1qoTq7qn6EYiU7qQgbNHglww014PT4O/3G5eo/3kbJu/M/IxxTyGw==
```

Comment:
@yakovkaravelov this will cause CI fails. Run our script to clean `yarn.lock` from integrity hashes. It is: `yarn lockfile:fix`

Reply:
@tomothespian OK.